src/components/global/Playground/index.tsx

```diff
@@ -38,7 +41,23 @@ export default function Playground() {
             </button>
           </div>
           <div className="playground__control-group playground__control-group--end">
-            {/* TODO FW-737: Toggle/Collapse Button */}
+            <button
+              className="playground__icon-button"
+              aria-label={expanded ? 'Hide source code' : 'Show full source'}
+              onClick={() => setExpanded(!expanded)}
+            >
+              <svg
```

Comment:
The rendered SVG looks very small compared to what's in the Figma design

Reply:
Updated 👍  was having an issue with that asset originally, but looks like Ben fixed it.